Zend/zend_vm_def.h

```diff
@@ -6688,6 +6688,22 @@ ZEND_VM_HANDLER(145, ZEND_DECLARE_INHERITED_CLASS_DELAYED, CONST, CONST)
 	ZEND_VM_NEXT_OPCODE_CHECK_EXCEPTION();
 }
 
+ZEND_VM_HANDLER(146, ZEND_VERIFY_VARIANCE, CONST, UNUSED)
+{
+	USE_OPLINE
+	zend_class_entry *zce;
+
+	SAVE_OPLINE();
+	zce = zend_lookup_class(Z_STR_P(RT_CONSTANT(opline, opline->op1)));
+	if (zce) {
+		zend_verify_variance(zce);
+	} else {
+		ZEND_ASSERT(EG(exception));
+		HANDLE_EXCEPTION();
+	}
```

Comment:
The code is modeled after `ZEND_DECLARE_INHERITED_CLASS` and `ZEND_DECLARE_INHERITED_CLASS_DELAYED`, which have code like this. I suppose this could mean that the class they depended on could not be found, but for `ZEND_VERIFY_VARIANCE` this should always work.